src/AssetControlExtension.php

```diff
@@ -209,6 +217,23 @@ protected function addAssetsFromRecord(AssetManipulationList $manipulation, Data
         }
     }
 
+    /**
+     * Check if the owner has assets
+     *
+     * @return bool
+     */
+    protected function hasAssets()
```

Comment:
This will need to be a private method so that it's not an API additin, which we can't do in 1.13 and we're not planning on releasing 1.14 now that CMS 5 is out

Reply:
i made it private. regarding the return type, that would be the only function with a strongly typed in the whole class, which i think is out of the scope. if the team wants to make specific style adjustements or syntax adjustement, i'll leave it up to you ;-)
i had a similar discussion here https://github.com/silverstripe/silverstripe-framework/pull/10834#pullrequestreview-1494413433 and i don't get why the pr should not follow the overall current style of the class. either the class use strong return type, or it doesn't. but mixing it doesn't feel right :-)

Reply:
When adding new methods, properties, etc, our preference is that they be strongly typed where possible.  

In general if a request like that is made you'll find less friction to having your pr be merged if you make the requested change - yes it's a little inconsistent with existing API, but it's also more robust.

We can't make everything strongly typed outside of a major release and we didn't have time to do that for CMS 5. We can add strong typing to new methods as we add them, though, so that's what we prefer to do. It will ultimately reduce the burden when we do go through with a comb and strongly type everything.

Reply:
ok ;-) works for me as well, i've updated the pr accordingly